app/helpers/application_helper.rb

```diff
@@ -505,10 +505,16 @@ def app_metadata
       UISettings: ui_settings, version: SETTINGS[:version].short, docUrl: documentation_url,
       location: Location.current&.attributes&.slice('id', 'title'),
       organization: Organization.current&.attributes&.slice('id', 'title'),
-      user: User.current&.attributes&.slice('id', 'login', 'firstname', 'lastname', 'admin')
+      user: user_metadata
     }.compact
   end
 
+  def user_metadata
+    return {} unless (user = User.current)
+    permissions = user.permissions.search_for('name ~ create_').pluck(:name)
```

Comment:
Do you mean that it lists no permissions for admin? 

![user-perm](https://user-images.githubusercontent.com/2664090/124719866-e2563e00-df07-11eb-8707-775efa1e8f6b.png)

As far as I know, admin does not need any permissions, because admins are allowed to do anything (and permissions are not checked in that case).